Account/FxAClient10.swift

```diff
@@ -212,6 +255,40 @@ public class FxAClient10 {
         return nil
     }
 
+    private class func statusResponseFromJSON(json: JSON) -> FxAStatusResponse? {
```

Comment:
Didn't write these methods, but they're following the convention of other methods in this class: if it can be statically defined, do so. I think it's semantically clearer since it reveals that these methods aren't tied to the class instance state (and won't have any side effects).